server/src/test-data/fixtures.js

```diff
@@ -22,25 +22,23 @@ export const DEFAULT_GROUP = 'NSW SES';
 
 export const ORG_NAME = 'NSW SES';
 
-export const CAPABILITIES = [
-  'Road Crash Rescue',
-  'Vertical Rescue',
-  'Flood Rescue',
-  'Chainsaw',
-];
 
 export const TAGS = [
```

Comment:
These are the capabilities that I received from Mel:


Qualification in SAP | Roll Up Name in SAP
-- | --
Storm Ground | Storm and Water Damage - Ground
Storm Heights | Storm and Water Damage - Heights
Flood Rescue Operator Level 1 | Flood Rescue Operator Level 1
Flood Rescue Operator Level 2 | Flood Rescue Operator Level 2
Flood Rescue Boat Operator | Flood Rescue Boat Operator
Flood Rescue Operator Level 3 | Flood Rescue Operator Level 3
GLR | General Land Rescue Operator
GLR2 | TBC
CFR | Community First Responder Registration
VR | Vertical Rescue Operator
Chainsaw (L1 Cross Cut) | Chainsaw L1 (Cross Cut)
Chainsaw (L2 Intermediate Felling) | Chainsaw L2 (Intermediate Felling)
Land Search 1 | Land Search

The following qualifications are yet to be determined:
 
·        Chainsaw (L3)
·        Aviation
·        Land Search 2
·        Land Search 3 (Alpine)
·        First Tier Training (Foundations) – Future State: Job Ready
·        Foundations – Future State: Job Ready


IMIC001 | SES IM Incident Controller
IMPO001 | SES IM Planning Officer
IMPIO001 | SES IM Public Information Officer
IMOO001 | SES IM Operations Officer
IMLO001 | SES IM Logistics Officer

---

The ones you have are OK for now but if we could put some IMT roles in it would be helpful.